qsim.cpp

```diff
@@ -270,6 +270,12 @@ const char** get_qemu_args(const char* kernel, int ram_size, int n_cpus, const s
     NULL
   };
 
+  /*
+  "-s",
+  "-S",
+  "-gdb", "tcp::1234",
+  */
+
```

Comment:
Yes, these lines are also not very necessary, but we have unsuccessfully try to debug a program in qemu in qsim.

If you like I can delete them.